### PR TITLE
Module Executor Telemetry Cleanup

### LINKED ISSUE #50: ModuleExecutor Contains Old Telemetry System
ModuleExecutor contains old telemetry code while also implementing TelemetryProvider, possibly making debugging difficult.

### PR DESCRIPTION
**Pull request by e3l**
Removed telemetry field variable from Module Executor and removed telemetryDump update from Robot. 

Resolves #50

### CODE REVIEW

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/rework/Robot.java [hunk 1]
```diff
@@ -68,14 +67,13 @@ public void update() {
                 try {
                     module.update();
                 }catch (Exception e){
-                    Log.d("Module", "Module couldn't update");
+                    Log.d("Module", "Module couldn't update: " + module.getClass().getSimpleName());
```
Comment (chop0) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/rework/Robot.java:70]: Is there a way you can get the name without using reflection?

**Review by chop0 — CHANGES_REQUESTED** [review]
Maybe add a getname thing, or something like that

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/rework/Robot.java (hunk 1 above)
Comment (e3l) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/rework/Robot.java:70]: @chop0 what's the problem with this? I think it's what you used for telemetry also

Comment (chop0) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/rework/Robot.java:70]: yes, but im bad. reflection allegedly slows stuff down, but its fine ig

**Review by chop0 — APPROVED** [review]
lgtm!

**Pull request merged**